airbyte-metrics/metrics-lib/build.gradle

```diff
@@ -9,10 +9,12 @@ dependencies {
     implementation project(':airbyte-db:db-lib')
 
     implementation libs.otel.semconv
+    implementation libs.otel.sdk
     implementation platform(libs.otel.bom)
     implementation("io.opentelemetry:opentelemetry-api")
     implementation("io.opentelemetry:opentelemetry-sdk")
     implementation("io.opentelemetry:opentelemetry-exporter-otlp")
+    implementation 'io.opentelemetry:opentelemetry-sdk-metrics-testing:1.13.0-alpha'
```

Comment:
Curious, does the bom cover the sdk-metrics dependency? can we remove the explicit version?

Reply:
No - I'm not sure what bom covers, but it won't compile (InMemoryMetricExporter not found)